Responsive design, Project #: 6 Request for Code review

wickathou opened the pull request:
Features:

+ CSS responsive rules
+ Clone of the TNW website

wickathou committed:
Initial commit:
+ Added HTML structure
+ Added bootstrap classes
+ Custom CSS rules
+ Dummy content added

wickathou committed:
Initial commit:
+ Added HTML structure
+ Added bootstrap classes
+ Custom CSS rules
+ Dummy content added

wickathou committed:
Changes:
+ Added hover rules
+ Updated responsiveness on the site
+ Updated HTML structure
+ Updated Readme

wickathou committed:
Changes:
+ Added hover rules
+ Updated responsiveness on the site
+ Updated HTML structure
+ Updated Readme

wickathou committed:
Change on the HTML head, updated title

wickathou committed:
Change on the HTML head, updated title

wickathou committed:
Repo rebased

wickathou committed:
Repo rebased

wickathou committed:
Merge branch 'structure' of github.com:wickathou/tnw-responsive into structure

wickathou committed:
updates to CSS lint

wickathou committed:
update to readme style

wickathou committed:
Changes for code lint, CSS

wickathou committed:
Changes for code lint, CSS

wickathou committed:
Changes for code lint, CSS

whiz25 reviewed (changes requested):
## Good job :+1: 
Your clone looks good. However, we need to make it better before we can merge.
- [x] make your navbar fixed
- [x] the latest news section should have three images per row on the screen size of ```768px``` and below.
- [x] the two images on the right of the top three articles section should have the same height as the one image on the left. Check the original site and fix accordingly.
![Screenshot from 2019-10-16 08-35-00](https://user-images.githubusercontent.com/19773024/66890905-4e846600-eff0-11e9-8263-3822c349a78d.png)
- [ ] remove comments from your code.

### Make the suggested changes and re-submit for another review.

wickathou committed:
Changes requested:
- Make navbar fixed > Done
- Latest news should have 3 articles at width 768px > Done
- he two images on the right of the top three articles section should have the same height as the one image on the left. Check the original site and fix accordingly > Done
- Remove comments from your code X Not required, these comments are used to help navigate the code

wickathou commented:
+ Navbar fixed
+ Latest news section features 3 articles at 768px
+ Cover images sizing fixed

**Comments in the code will not be erased. Their purpose is to organize the code**

Redvanisation reviewed (approved):
### Hello @wickathou 

### _Nice work on this project, it seems that you guys have met the requirements so I am approving your project. 👍_ 

### You may merge to master and submit a project completion form. 🚀 

### Good luck! 🍀
